Scripts/DCS-BIOS/lib/modules/aircraft_modules/FW-190D9.lua

```diff
@@ -0,0 +1,224 @@
+FW_190A8:defineFloat("COMPASSHEADING", 45, { 0, 1 }, "Indicator", "CompassHeading")
+FW_190A8:defineFloat("COMMANDEDCOURSE", 44, { 0, 1 }, "Indicator", "CommandedCourse")
+FW_190A8:defineFloat("MANIFOLD_PRESSURE", 46, { 0, 1 }, "Indicator", "Manifold_Pressure")
+FW_190A8:defineFloat("ENGINE_RPM", 47, { 0, 0.983 }, "Indicator", "Engine_RPM")
+FW_190A8:defineFloat("COOLANT_TEMPERATURE", 96, { 0.07, 1 }, "Indicator", "Coolant_Temperature")
+FW_190A8:defineFloat("OIL_TEMPERATURE", 97, { 0.07, 1 }, "Indicator", "Oil_Temperature")
+FW_190A8:defineFloat("OIL_PRESSURE", 95, { 0, 1 }, "Indicator", "Oil_Pressure")
+FW_190A8:defineFloat("FUEL_PRESSURE", 94, { 0, 1 }, "Indicator", "Fuel_Pressure")
+FW_190A8:defineFloat("MW50_PRESSURE", 106, { 0, 1 }, "Indicator", "MW50_Pressure")
+FW_190A8:defineFloat("FUELMETER", 98, { 0, 1 }, "Indicator", "FuelMeter")
+FW_190A8:defineFloat("FLUOR_LIGHT", 195, { 0, 1 }, "Indicator", "Fluor_Light (green)")
+FW_190A8:defineFloat("PILOT_DRAW", 540, { 0, 1 }, "Indicator", "pilot_draw")
+FW_190A8:defineFloat("CURRTIME_HOURS", 21, { 0, 1 }, "Indicator", "CurrTime_Hours")
+FW_190A8:defineFloat("CURRTIME_MINUTES", 22, { 0, 1 }, "Indicator", "CurrTime_Minutes")
+FW_190A8:defineFloat("CURRTIME_SECONDS", 23, { 0, 1 }, "Indicator", "CurrTime_Seconds")
+FW_190A8:defineFloat("CHRONO_MINUTES", 27, { 0, 1 }, "Indicator", "Chrono_Minutes")
+FW_190A8:defineFloat("AMMO_COUNTER_1", 52, { 0, 1 }, "Indicator", "Ammo_Counter_1")
+FW_190A8:defineFloat("AMMO_COUNTER_2", 55, { 0, 1 }, "Indicator", "Ammo_Counter_2")
+FW_190A8:defineFloat("AMMO_COUNTER_3", 58, { 0, 1 }, "Indicator", "Ammo_Counter_3")
+FW_190A8:defineFloat("AMMO_COUNTER_4", 61, { 0, 1 }, "Indicator", "Ammo_Counter_4")
+FW_190A8:defineFloat("TARGETDIST", 133, { 0, 1 }, "Indicator", "TargetDist")
+FW_190A8:defineFloat("ROCKETEMCVR", 137, { 0, 1 }, "Indicator", "RocketEmCvr")
+FW_190A8:defineFloat("B_1_KNOPF", 9, { 0, 1 }, "Indicator", "B_1_knopf")
+FW_190A8:defineFloat("B_2_KNOPF", 11, { 0, 1 }, "Indicator", "B_2_knopf")
+FW_190A8:defineFloat("COVER_KNOPF", 10, { 0, 1 }, "Indicator", "Cover_knopf")
+FW_190A8:defineFloat("TRIMMNEEDLE", 76, { -1, 1 }, "Indicator", "TrimmNeedle")
+FW_190A8:defineFloat("ADF_VERTICAL", 29, { -1, 1 }, "Indicator", "ADF_Vertical")
+FW_190A8:defineFloat("ADF_HORIZONT", 30, { -1, 1 }, "Indicator", "ADF_Horizont")
+FW_190A8:defineFloat("WINDSHIELDDAMAGE", 209, { 0, 1 }, "Indicator", "WindshieldDamage")
+FW_190A8:defineFloat("CANOPYDAMAGE", 210, { 0, 1 }, "Indicator", "CanopyDamage")
+
+FW_190A8:defineFixedStepTumb("FUG16_TUNING", 15, 3008, 82, 0.01, { -1, 1 }, { -0.01, 0.01 }, nil, "Radio", "Radio Tuning (+/-30 kHz)")
+FW_190A8:defineTumb("STARTER_SWITCH2", 2, 3009, 105, 0.5, { 0, 1 }, nil, false, "Cockpit", "Starter Switch. Magnetic Clutch")
+FW_190A8:define3PosTumb("FUEL_COLD_START", 6, 3007, 160, "Cockpit", "Cold Start/Wind Screen Washer")
+
+--Externals
+FW_190A8:defineIntegerFromGetter("EXT_POSITION_LIGHT_LEFT", function()
```

Comment:
Module:defineBitFromDrawArgument can be used to clean up these external calls

Reply:
done